lms/models/lti_registration.py

```diff
@@ -0,0 +1,35 @@
+import sqlalchemy as sa
+
+from lms.db import BASE
+from lms.models import CreatedUpdatedMixin
+
+
+class LTIRegistration(CreatedUpdatedMixin, BASE):
```

Comment:
I think having `LTI` in the name might be bit verbose but makes clearer what this is about.

Just `registration` or something like that might be too vague.

Reply:
I think it's good. It makes it clear it's not our concept and choice of name.